src/lib/components/auth-cards/CreateCard.svelte

```diff
@@ -107,21 +116,27 @@
 
   const setMessage = () => {
     let emailAlreadyExist = "auth/email-already-in-use";
+    let invalidEmail = "auth/invalid-email"
     let isExistingEmail;
+    let isInvalidEmail
```

Comment:
Shouldn't this have a semi-colon? I wonder why eslint isn't pointing this out.

Reply:
Huh, that's so weird. Eslint works it looks like I forgot to format it. So it was my mistake.